tests/components/gree/test_climate.py

```diff
@@ -222,17 +224,16 @@ async def test_discovery_device_bind_after_setup(
     MockDevice1.bind.side_effect = None
     MockDevice1.update_state.side_effect = None
 
-    next_update = mock_now + timedelta(minutes=5)
-    freezer.move_to(next_update)
-    async_fire_time_changed(hass, next_update)
+    freezer.tick(timedelta(seconds=DISCOVERY_SCAN_INTERVAL) + timedelta(minutes=1))
```

Comment:
The extra minute should not be needed, but if it doesn't work otherwise, it's not a blocker